pkg/ocibundle/ocisif test dependent on squashfuse, but not gated

This is a long test, skipped in CI short-unit-tests

```
$ sudo go test -count 1 ./pkg/ocibundle/ocisif/...
--- FAIL: TestFromImageRef (1.21s)
    --- FAIL: TestFromImageRef/oci-sif (0.00s)
        bundle_linux_test.go:52: While creating bundle: while mounting squashfs layers: while mounting squashfs layer: use of image "/tmp/oci-test331009153" as overlay requires squashfuse to be installed: exec: "squashfuse": executable file not found in $PATH
        oci.go:47: Could not create bundle validator: open /tmp/TestFromImageRefoci-sif3703418552/001/config.json: no such file or directory
            Refer to: https://github.com/opencontainers/runtime-spec/blob/v1.2.0/bundle.md#container-format
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x855756]
```

--- a/pkg/ocibundle/ocisif/bundle_linux_test.go
+++ b/pkg/ocibundle/ocisif/bundle_linux_test.go
@@ -49,7 +49,7 @@ func TestFromImageRef(t *testing.T) {
 			}
 
 			if err := b.Create(context.Background(), nil); err != nil {
-				t.Errorf("While creating bundle: %s", err)
+				t.Fatalf("While creating bundle: %s", err)
 			}
 
 			ocitest.ValidateBundle(t, bundleDir)

